testsuite/cluster-test/src/experiments/performance_benchmark_three_region_simulation.rs

```diff
@@ -38,26 +38,56 @@ impl ExperimentParam for PerformanceBenchmarkThreeRegionSimulationParams {
 #[async_trait]
 impl Experiment for PerformanceBenchmarkThreeRegionSimulation {
     async fn run(&mut self, context: &mut Context<'_>) -> anyhow::Result<()> {
-        let (us, euro) = self.cluster.split_n_validators_random(80);
-        let (us_west, us_east) = us.split_n_validators_random(40);
-        let network_effects = three_region_simulation_effects(
-            (
-                us_west.validator_instances().to_vec(),
-                us_east.validator_instances().to_vec(),
-                euro.validator_instances().to_vec(),
-            ),
-            (
-                Duration::from_millis(60), // us_east<->eu one way delay
-                Duration::from_millis(95), // us_west<->eu one way delay
-                Duration::from_millis(40), // us_west<->us_east one way delay
-            ),
-        );
+        let (us, euro) = if context.is_fullnode {
```

Comment:
I think all this part should rename the same - we care about splitting validators between regions regardless of where we submit transactions to